thirdparty/patches/opencensus-cpp-shutdown-api.patch

```diff
@@ -5,7 +5,7 @@ index 3bb5962..706b9b4 100644
 @@ -25,7 +25,7 @@ namespace opencensus {
  namespace exporters {
  namespace stats {
- 
```

Comment:
why are there logic changes in this file?

Reply:
No logic change, only `GUARDED_BY` -> `ABSL_GUARDED_BY`